Dockerfile

```
@@ -596,3 +596,94 @@ USER nextjs
 # server.js is created by next build from the standalone output
 # https://nextjs.org/docs/pages/api-reference/next-config-js/output
 CMD ["node", "apps/vpnmanager/server.js"]
+
+
+
+# ============================================================================
+# Climate Mapped Africa
+# ============================================================================
+
+#
+# climatemapped-africa-desp: image with all climatemapped-africa dependencies
+# ---------------------------------------------------
+
+  FROM base-deps as climatemapped-africa-deps
```

```suggestion
  FROM base-deps as climatemappedafrica-deps
```

App name should just be one word: charterafrica, codeforafrica, vpnmanager, etc.